src/template.ts

```diff
@@ -262,7 +264,7 @@ canvas.show {
         will have their <slot> elements removed by ShadyCSS -->
   <div class="slot poster">
     <slot name="poster">
-      <div id="default-poster" aria-hidden="true" aria-label="Activate to view in 3D!"></div>
+      <button type="button" id="default-poster" aria-hidden="true" aria-label="Activate to view in 3D!"></button>
```

Comment:
`aria-hidden="true"` hides this element from assistive technology (yet the tab-stop remains.) Please remove the `aria-hidden` attribute.

Reply:
@svinkle the poster starts out in a hidden state unless the user configures it to be visible. If we don't do this, then it is always possible for the user to tab through the poster even when it is not configured or otherwise visible.

Reply:
@cdata Got it. Noticed this when reading through [`$showPoster()`](https://github.com/GoogleWebComponents/model-viewer/blob/028176b90006a90595b67f77a2d094c85a684113/src/features/loading.ts#L346).